src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -285,21 +373,41 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
   }
 }
 
+PipelineRuns.defaultProps = {
+  filters: []
+};
+
+export function fetchFilters(props) {
```

Comment:
should just accept the search string as param, it shouldn't need to access anything else from either props or location